spec/harness/active_publisher_spec.rb

```diff
@@ -1,11 +1,46 @@
 require "spec_helper"
 
-RSpec.describe Harness::ActivePublisher do
+describe ::Harness::ActivePublisher do
+  let(:collector) { ::Harness::NullCollector.new }
+
+  before do
+    ::Harness.config.queue = ::Harness::SyncQueue.new
+    ::Harness.config.collector = collector
+  end
+
   it "has a version number" do
     expect(Harness::ActivePublisher::VERSION).not_to be nil
   end
 
-  it "does something useful" do
-    expect(false).to eq(true)
+  describe "async_queue_size.active_publisher" do
+    it "updates the queue size count" do
+      expect(collector).to receive(:count).with("active_publisher.async_queue_size", 1000)
```

Comment:
I think we might want to report this as a `gauge` since it represents a sampled value rather than a change. https://github.com/etsy/statsd/blob/master/docs/metric_types.md#gauges